src/state/BoardConstructor.js

```diff
@@ -0,0 +1,52 @@
+import { ROWS, COLUMNS } from "../constants";
+import { EMPTY, SCARAB, PHARAOH, ANUBIS, PYRAMID } from "./BoardBoss";
+
+export default class BoardConstructor {
+  constructor(gameType = "classic") {
+    const buildEmptyBoard = () => {
+      let emptyBoard = [];
+
+      for (let r = 0; r < ROWS; r++) {
+        let newRow = [];
+
+        for (let c = 0; c < COLUMNS; c++) {
+          newRow.push(EMPTY);
+        }
+
+        emptyBoard.push(newRow);
+      }
+
+      return emptyBoard;
+    };
+
+    this.board = buildEmptyBoard();
+
+    switch (gameType) {
+      case "classic":
```

Comment:
This is based on the Classic configuration. There are 2 other "OG" configurations (Imhotep and Dynasty). We'll stick with Classic for now.